db/seeds.rb

```diff
@@ -14,28 +14,7 @@
     user.save
 end 
```

Comment:
Oy, don't you need a few other things to create an user? Like password?!

Reply:
sure